pyiron_atomistics/sphinx/output_parser.py

```diff
@@ -391,17 +391,25 @@ def get_fermi(self):
         return np.array(re.findall(pattern, self.log_main)).astype(float)
 
 
-class SphinxWavesParser():
+class SphinxWavesParser:
     """ Class to read SPHInX waves.sxb files (HDF5 format)
     
         Initialize with waves.sxb filename, or use load ()
     
     """
     
-    def __init__(self, fname = None):
-        if (fname is not None):
-            self.load (fname)
-    
+    def __init__(self, file_name="waves.sxb", cwd="."):
+        """
+        Args:
+            file_name (str): file name
+            cwd (str): directory path
+        """
+        if cwd is None:
+            cwd = "."
```

Comment:
`if cwd is None` etc. is included in the other parsers for the backward compatibility. Here in this class I don’t think it’s required, because it’s new

Reply:
coolio